openapi/v2/core-metadata.yaml

```diff
@@ -2454,15 +2499,15 @@ paths:
                 500Example:
                   $ref: '#/components/examples/500Example'
     patch:
-      summary: "Allows device command updates to an existing device service, such as command's readWrite and isHidden fields."
+      summary: "Allows device command updates to an existing device service, such as command's isHidden fields."
```

Comment:
summary: "Allows the isHidden field of the existing device commands to be updated."